Microsoft.DotNet.Try.Jupyter/ExecuteRequestHandler.cs

```diff
@@ -96,13 +96,13 @@ void IObserver<IKernelEvent>.OnNext(IKernelEvent value)
             }
         }
 
-        private void OnCodeSubmissionEvaluatedFailed(CodeSubmissionEvaluationFailed codeSubmissionEvaluationFailed, ConcurrentDictionary<Guid, OpenRequest> openRequests)
+        private static void OnCodeSubmissionEvaluatedFailed(CodeSubmissionEvaluationFailed codeSubmissionEvaluationFailed, ConcurrentDictionary<Guid, OpenRequest> openRequests)
         {
             var openRequest = openRequests[codeSubmissionEvaluationFailed.ParentId];
  
             var errorContent = new Error(
                 eName: "Unhandled Exception",
-                eValue: $"{codeSubmissionEvaluationFailed.Error}"
+                eValue: $"{codeSubmissionEvaluationFailed.Message}"
```

Comment:
what will happen to the "Error" property now. Doesnt it have any useful information ?

Reply:
Error is an object and this this case we use the message (if message is not set then object to string is used or the message field is the error is an exception